tests/test.collector.js

```diff
@@ -65,4 +65,31 @@ describe('Collector', function() {
     assert.equal(collection.notices.length, 0);
   });
 
+  it('should filter message by filename if config.scanFile is defined', () => {
```

Comment:
Can you just test the actual code of the errors in the collection so we know the right ones are getting through? Thanks ^_^